### ISSUE TITLE
Scripts for generating different-dim embeddings

### ISSUE THREAD

**Issue by Liu-rj** [IllinoisGraphBenchmark/IGB-Datasets#34]
Hi,

I'm very interested in the IGB graphs with different-dim node embeddings and the IGB paper says there are downloadable embeddings from the RoBERTa language model and has a toolkit to generate other embeddings. But sadly it seems that I do not find any related contents in the current codebase. So could you explicitly give provide the data sources or give some experimental scripts on achieving this?

Any help or guidance is appreciated.

**Comment by akhatua2** [comment]
Hi @Liu-rj we used a naive dimensionality reduction script using [cuML PCA](https://docs.rapids.ai/api/cuml/nightly/api/). Or you can use this follow the sklearn's PCA implementation in this [line](https://github.com/IllinoisGraphBenchmark/IGB-Datasets/blob/1bc83b276e37ca59a4a1cfcb648cc55ab1f7d245/igb/dataloader.py#L48-L49):    

```bash
pca = PCA(n_components=x) # check out additional parameters from https://scikit-learn.org/stable/modules/generated/sklearn.decomposition.PCA.html#sklearn.decomposition.PCA.fit_transform
emb  =  pca.fit_transform(emb)
return emb
```


 In regards to the model-generated embedding with variable dimensions, unfortunately, it requires a huge amount of s3 space and we don't currently have the allocation to share it.

**Comment by Liu-rj** [comment]
Thx for informing that, I'll try it out.
